onmt/translate/translator.py

```diff
@@ -137,6 +138,8 @@ def __init__(
         self._tgt_pad_idx = self.vocabs['tgt'].lookup_token(DefaultTokens.PAD)
         self._tgt_bos_idx = self.vocabs['tgt'].lookup_token(DefaultTokens.BOS)
         self._tgt_unk_idx = self.vocabs['tgt'].lookup_token(DefaultTokens.UNK)
+        self._tgt_start_with =\
```

Comment:
Is it expected we retain _tgt_bos_idx here, when we add _tgt_start_with?

Reply:
bos remains. we just added a new parameter to make the decoder start token different in some cases (eg: FSQ models use eos as the decoder start token, no clue why)